code/modules/mining/ores_coins.dm

```diff
@@ -52,12 +52,18 @@
 /obj/item/stack/ore/burn()
 	if(!refined_type)
 		return ..()
-	var/obj/item/stack/sheet/S = new refined_type(drop_location())
-	var/percent = rand(0.3,0.7)
-	var/amountrefined = round(amount*percent)

```

Comment:
@fludd12 btw this is what was causing the issue. round acts as a floor if you don't specify what to round to

See http://www.byond.com/docs/guide/chap16.html

Reply:
oh flip sorry